docs/antctl.md

```diff
@@ -781,6 +781,34 @@ PEER               ASN   STATE
 192.168.77.201:179 65002 Active
 ```
 
+`antctl` agent command `get bgproutes` prints the advertised BGP routes.
```

Comment:
Probably better to highlight it prints the routes on the local Node only.

```suggestion
`antctl` agent command `get bgproutes` prints the advertised BGP routes on the local Node.
```

Reply:
done, thanks @luolanzone